ProgramParsing/Science/MajorParser2021_2022.py

```diff
@@ -85,6 +86,18 @@ def _course_list(self, line, credit, oneOf = False):
             for c in courses:
                 if c not in d:
                     list.append(c)
+        
+        # handles: 1.25 BIOL units including BIOL 130, BIOL 130L, and 0.5 BIOL unit at 100- or 200-level (exclusive of BIOL 225 and BIOL 280)
+        if ("Psychology" in program_name and ("BIOL 225" in list or "BIOL 280" in list)):
```

Comment:
Is it safer to use` "psychology" in program_name.lower() `?

Reply:
Sorry I just saw this, I will fix this in the next PR